litestar/contrib/sqlalchemy/repository/_async.py

```diff
@@ -245,8 +271,13 @@ async def get(  # type: ignore[override]
         """
         with wrap_sqlalchemy_exception():
             id_attribute = id_attribute if id_attribute is not None else self.id_attribute
-            statement = statement if statement is not None else self.statement
-            statement = self._filter_select_by_kwargs(statement=statement, kwargs=[(id_attribute, item_id)])
+            statement = self._to_lambda_stmt(statement)
+            statement = self._filter_select_by_kwargs(
+                statement=statement,
+                kwargs={
+                    id_attribute: item_id,  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
pyright is actually right here. `id_attribute` is not guaranteed to be hashable. That's why I added support for passing this as an iterable of tuples.

Reply:
Is there any action to take on this one?  I wasn't exactly sure.